### PR TITLE
Header redesign

### PR DESCRIPTION
**Pull request by tgandrews**
### What is the context of this PR?
Header redesign as per prototype.

Interesting changes
1. There are now 2 headers - one defined in EditorLayout and one for the non editor pages in Layout
1. I have completely removed the properties panel from EditorLayout as it is a question page thing and not generic to all pages in the editor.

### How to review 
1. Ensure it matches the prototype `prototype-new-header-nav-2`
1. Ensure that everything works as expected

### CODE REVIEW

**Comment by hamishtaplin** [comment]
> Page is scrolling up when adding new confirmation/calculated summary/section page. It then won't allow user to scroll back to top of page.
> 
> To reproduce:
> 
> 1. Create new questionnaire
> 2. Immediately create Section

I had this issue in the prototype and I fixed it but I can't remember how. It was a CSS problem though.

**Comment by tgandrews** [comment]
@hamishtaplin I fixed it applying an overflow hidden to the grid. It wasn't showing the scrollbar for the editor content. https://github.com/ONSdigital/eq-author-app/pull/447/commits/12afa55704f9c24df02831f4d42d5093bbb01c13

File: eq-author/src/App/QuestionnaireSettingsModal/index.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 import PropTypes from "prop-types";
-import QuestionnaireMeta from "./QuestionnaireMeta";
-import CustomPropTypes from "custom-prop-types";
 import { noop } from "lodash";
 import gql from "graphql-tag";
 
+import CustomPropTypes from "custom-prop-types";
+
+import DialogHeader from "components/Dialog/DialogHeader";
+import Modal from "components/modals/Modal";
```
[the remark this answers is not in the document]
Comment (tgandrews) [reply eq-author/src/App/QuestionnaireSettingsModal/index.js]: No I'll revert it. I think I made changes on the file and tidied up the imports whilst I was there but removed the change. Good spot

**Comment by tgandrews** [comment]
[Redesign of header (6)](https://trello.com/c/Ca1jYp4C/1148-redesign-of-header-6)

**Pull request merged**